modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+
+[id="known-issues-1-16_{context}"]
+== Known issues
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16 and later.
+
+[id="fixed-issues-1-16_{context}"]
+== Fixed issues
+
+* Before this update, when you uninstalled {tekton-hub} by deleting the `TektonHub` CR, the pod of the `hub-db-migration` job was not deleted. With this update, uninstalling {tekton-hub} deletes the pod.
+
+* Before this update, when you used {tekton-results} to store pod logs from pipelines and tasks, the operation to store the logs sometimes failed. The logs would include the `UpdateLog` action failing with the `canceled context` error. With this update, the operation completes correctly. 
+
+* Before this update, when you passed a parameter value to a pipeline or task and this value included more than one variable, and they use both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly; the pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
+
+* Before this update, {tekton-chains} failed to generate correct attestations when a task run included multiple tasks with the same name. For instance, when using a matrix of tasks, the attestation was generated for the first image. With this update, {tekton-chains} generates attestations for all tasks within the task run, ensuring complete coverage.
+
+* Before this update, when you used the `skopeo-copy` task defined in the {pipelines-shortname} installation namespace and set its `VERBOSE` parameter to `false`, the task failed. With this update, the task completes normally.
+
+* Before this update, if you installed {pipelines-shortname} on a cluster with a high number of namespaces, the installation could take a significant time, because the Operator created RBAC resources in every namespace in a sequence. With this update, the Operator creates the RBAC resources concurrently, resulting in a faster installation.
+
+* Before this update, when using {pac}, if you set the `concurrency_limit` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` or `pipelines-as-code` namespace, which provides default settings for all `Repository` CRs, the {pac} watcher crashed. With this update, the {pac} watcher operates correctly with this setting.
+
+* Before this update, the `buildah`, `s2i-*`, and `skopeo-copy` tasks included an extra step compared to the cluster tasks of the same name that were available in previous versions of {pipelines-shortname}. This extra step increased the load on the cluster. With this update, the tasks no longer include the extra step as it is integrated into the first step.
```

Comment:
I think all the ecosystem tasks now have single steps similar to clusterTask not just the `buildah`, `s2i-*`, and `skopeo-copy` cc @aneeshmbhat 
can you give details to @mramendi so that he can update it

Reply:
yes correct, you can just mention all tasks that are part of openshift-pipelines, since all of them are changed to reflect so.
```

( removed 1 load-scripts step thats all )
cases where we only removed load-scripts step & added the logic to the next step are:
tkn
all s2i tasks
openshift-client
opc
maven
buildah

( along with load-scripts step, removed other steps, since they all used the same image )
Some cases where we removed multiple steps along with load-scripts are:
git-clone -> reduced to single step from 4 steps
git-cli -> reduced to single step from 3 steps
skopeo-copy -> reduced 3 steps to single step
```

Reply:
Changed it to `Before this update, all tasks in {pipelines-shortname} included an extra step....`
Let me know if that is not correct. Thank you!

Reply:
Hello @Aneesh-M-Bhat I am not sure I follow now. I changed it from "buildah, s2i & skopeo-copy" to "all tasks", so what exactly is incorrect now? I left the rest of the RN, just changed buildah, s2i & skopeo-copy to "all tasks". I hope now it is clearer. Thank you!

Reply:
`Before this update, all tasks in {pipelines-shortname} included an extra step....`
this should be fine @eromanova97 
but the change wasn't reflected when i saw, but if u change it to this, thats good yes.